go.mod

```diff
@@ -123,6 +123,7 @@ require (
 	github.com/Azure/azure-sdk-for-go/sdk/internal v1.11.1 // indirect
 	github.com/AzureAD/microsoft-authentication-library-for-go v1.4.2 // indirect
 	github.com/alecthomas/units v0.0.0-20240927000941-0f3dac36c52b // indirect
+	github.com/andybalholm/brotli v1.2.0 // indirect
```

Comment:
where is this coming from? it's indirect but no other dep is changing

Reply:
it's coming from CH driver, we probably didn't import all the same modules previously
```
$ go mod graph | grep andybalholm/brotli
github.com/jaegertracing/jaeger github.com/andybalholm/brotli@v1.2.0
github.com/ClickHouse/clickhouse-go/v2@v2.40.1 github.com/andybalholm/brotli@v1.2.0
github.com/andybalholm/brotli@v1.2.0 github.com/xyproto/randomstring@v1.0.5
github.com/andybalholm/brotli@v1.2.0 go@1.22
```